src/towncrier/build.py

```diff
@@ -193,7 +193,7 @@ def __main(
         start_string = config["start_string"]
         news_file = config["filename"]
 
-        if config["single_file"]:
+        if not config["single_file"]:
             # When single_file is enabled, the news file name changes based on the version.
```

Comment:
Is this change ok ?

I was expecting something like this

```suggestion
        if config["single_file"] is True:
            # When single_file is enabled,
            # the release notes for each version are stored in a separate file.
            # The name of that file is generated based on the current version and project.
```